src/ripple/app/misc/ValidatorList.h

```diff
@@ -29,9 +29,8 @@
 #include <ripple/protocol/PublicKey.h>
 #include <boost/iterator/counting_iterator.hpp>
 #include <boost/range/adaptors.hpp>
-#include <boost/thread/locks.hpp>
-#include <boost/thread/shared_mutex.hpp>
 #include <mutex>
```

Comment:
I think `<mutex>` can be removed

Reply:
Actually, I think it wasn't needed before these changes. So now removal counts as bonus points.